pontoon/machinery/utils.py

```diff
@@ -71,9 +71,11 @@ def get_concordance_search_data(text, locale):
     )
     search_query = reduce(operator.and_, search_filters)
 
-    search_query_results = base.models.TranslationMemoryEntry.objects.filter(
-        search_query
-    ).values_list("source", "target", "project__name")
+    search_query_results = (
+        base.models.TranslationMemoryEntry.objects.filter(search_query)
+        .values_list("source", "target", "project__name")
```

Comment:
While we're at it, maybe also add these two fields?
https://github.com/mozilla/pontoon/pull/1794#discussion_r549899420